src/Reader/Reader.php

```diff
@@ -235,7 +241,7 @@ public static function import($uri, $etag = null, $lastModified = null)
             if ($data) {
                 return static::importString($data);
             }
-            $response = $client->send();
+            $response = $client->get($uri);
```

Comment:
Get is defined in ClientInterface, which is what we typehint on; there's nothing unsafe here at all. I think you're confusing this with HeaderAwareClHeaderAwareCli:getHeaderLine. (This comment applies to your other notices about this as well.)